tools/docker/Dockerfile.monorepo

```diff
@@ -38,14 +40,17 @@ RUN --mount=type=cache,target=/var/lib/apt/lists/,sharing=private \
         docker-compose-plugin \
         jq \
         "php${PHP_VERSION}" \
+        "php${PHP_VERSION}-ast" \
         "php${PHP_VERSION}-cli" \
         "php${PHP_VERSION}-curl" \
         "php${PHP_VERSION}-dom" \
         "php${PHP_VERSION}-mbstring" \
         "php${PHP_VERSION}-xml" \
         "php${PHP_VERSION}-zip" \
         rsync \
+    && apt-get remove --purge -y python3-apt \
     && apt-get remove --purge --auto-remove -y gpg software-properties-common \
+    && apt-get clean \
```

Comment:
I think the Ubuntu base image already disables what `apt-get clean` cleans. So this should wind up as a no-op.

Yeah, looks like that's in `/etc/apt/apt.conf.d/docker-clean`.

Reply:
cool, I'll remove that in a follow-up PR.